ideascube/monitoring/templates/monitoring/stock.html

```diff
@@ -15,7 +15,7 @@
         {% for item in module.objects  %}
             <tr class="stockitem" id="stockitem-{{ item.pk }}"><td>{{ item }} <em>{{ item.description|truncatewords:10 }}</em></td><td>{% trans 'Total:' %} {{ item.specimens.physical|length }}</td><td class="actions"><a href="{% url 'monitoring:stockitem_update' pk=item.pk %}" title="{% trans 'Edit' %}">{% fa 'pencil-square' %}</a> <a href="{% url 'monitoring:specimen_create' item_pk=item.pk %}" title="{% trans 'Add new specimen of this item' %}">{% fa 'plus-square' %}</a></td></tr>
             {% for specimen in item.specimens.physical %}
-                <tr class="specimen"><td>{{ specimen.barcode|default:'—' }}</td><td>{{ specimen.count }}</td><td class="actions"><a href="{% url 'monitoring:specimen_update' pk=specimen.pk %}" title="{% trans 'Edit this specimen' %}">{% fa 'pencil-square' %}</a></td></tr>
+                <tr class="specimen"><td>{{ specimen.pretty_identifier|default:'—' }}</td><td>{{ specimen.count }}</td><td class="actions"><a href="{% url 'monitoring:specimen_update' pk=specimen.pk %}" title="{% trans 'Edit this specimen' %}">{% fa 'pencil-square' %}</a></td></tr>
```

Comment:
I really think it would be better to display both the `barcode` and the `serial` here, just like in #659.